index.rst

```diff
@@ -11,14 +11,78 @@ Bharat-Sim's documentation!
    :caption: Contents:
 
 
-
-Indices and tables
-==================
-
 * :ref:`genindex`
 * :ref:`modindex`
 * :ref:`search`
 
-How does this lool?
+
+
+
+SIR Model with Agents
+=====================
+
+The Bharat-sim framework allows you to build agent-based models. Agents, are essentially the main component of the simulation. They are entities which are defined with a bunch of parameters and certain rules which they have to obey as the simulation runs.
+
+In the purview of the SIR Model, the agents are people, which go about their daily lives and follow regular schedules. Here, the parameters defining an agent are a unique agent id, age, their infection status, the time for which they remain infected and so on. The simulation allows you to introduce new parameters as and when required.
+
+The rules governing the behaviour of all agents include the particular schedules which they follow and, the functions which allow the agents to transit from one compartment to the other.
+
+Schedules
+---------
+
+The current schedules of all the agents are stored in the **create12HourSchedules** function. The default function has three schedules:-
+
+1. **employeeSchedule** - This schedule is followed by all agents above the age of 25.
```

Comment:
You are not describing what a schedule is clearly, and neither are you explaining the code here. Either you can avoid talking about schedules too much by referring to the "Framework Basics" section, but then you need to describe the lines in the code that make sure that all agents above the age of 25 follow this schedule. Furthermore, this part is incomplete without discussing the priority of schedules. Lastly, when you are describing something code related, don't just put it in bold. Use the appropriate markdown. You could even just use two of these quotes: `, for example, ``studentSchedule``, that shows you're talking about code.

Reply:
Yeah, sure I will elaborate on that section. I couldn't figure out how to do inline code so I just made it bold for now, thinking I will change it later

Reply:
Good. I would strongly suggesting figuring out all of these tiny things _first_, since going over your document and changing them one by one becomes _very_ tedious, and it's likely that you'll be less keen to change stuff like this everywhere later.